README.md

```diff
@@ -37,8 +37,11 @@ To learn more, check out the following resources:
 | [Contribution Documentation](./guides/CONTRIBUTING.md) | Learn how to contribute to DS-25. Find detailed instructions for installing, running, and working with the project locally.             |
 | [Architecture Documentation](./guides/ARCHITECTURE.md) | Explore the architectural decisions and principles behind DS-25. This includes explanations of project structure, benchmarks, and more. |
 
+## 4. Caveats
+For the time being, node 23 seems to provoke [an error](./issues/226). Use node v22 for the time being, for instance with `nvm use 22`.
```

Comment:
This issue link is a dead link when viewed directly in github
https://github.com/canonical/ds25/blob/c716db422ecf2fe1a2f3c5d5ebfd892c539b2ed3/README.md

It may be worth using an absolute URL.